tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -697,6 +697,27 @@ Then(
   }
 )
 
+When('the user tries to rename file {string} to {string}', function (fromName, toName) {
```

Comment:
We have similar step here https://github.com/owncloud/web/blob/f9779f30d472aebfc5d7e4755d749f99e4f9a4f6/tests/acceptance/stepDefinitions/filesContext.js#L344 
if this rename is through some other way please add something like `using the context menu`